istioctl/cmd/istioctl/authn.go

```diff
@@ -23,32 +25,35 @@ import (
 
 func tlsCheck() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "tls-check [<service>]",
+		Use:   "tls-check <pod-name[.namespace]> [<service>]",
 		Short: "Check whether TLS setting are matching between authentication policy and destination rules",
 		Long: `
-Requests Pilot to check for what authentication policy and destination rule it uses for each service in
-service registry, and check if TLS settings are compatible between them.
+Check what authentication policies and destination rules pilot uses to config a proxy instance,
+and check if TLS settings are compatible between them.
 `,
 		Example: `
-# Check settings for all known services in the service registry:
```

Comment:
I feel the old command that prints all known services is pretty useful to get an overview of the TLS status in the mesh, is it possible to have the same support in this command? Technically we could wrap this command in a script that queries every pod in the mesh.

Reply:
It could be too much, for example if some services have hundreds of replicas. Customer can still be able to do such a thing if they want with simple bash script.
A better option could be showing a svc-2-svc grid, but would be a bigger change and I don't think it is really needed at this point.